servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcServerBuilder.java

```diff
@@ -273,6 +274,23 @@ public final ServerContext listenAndAwait(GrpcServiceFactory<?, ?, ?>... service
         return awaitResult(listen(serviceFactories).toFuture());
     }
 
+     /**
+      * Starts this server and returns the {@link ServerContext} after the server has been successfully started.
+      * <p>
+      * If the underlying protocol (eg. TCP) supports it this will result in a socket bind/listen on {@code address}.
+      *
+      * @param services {@link GrpcService}(s) to create a <a href="https://www.grpc.io">gRPC</a> service.
+      * @return A {@link ServerContext} by blocking the calling thread until the server is successfully started or
+      * throws an {@link Exception} if the server could not be started.
+      * @throws Exception if the server could not be started.
+      */
+     public final ServerContext listenAndAwait(GrpcBindableService<?, ?, ?>... services) throws Exception {
```

Comment:
we should also add the async listen variant of the method (e.g. `Single<ServerContext> listen(GrpcBindableService<?, ?, ?>... services)`)

Reply:
Good idea. Added